### ISSUE TITLE
Review transitive include of e.g. "lib/vxlapi.h" in CAN_Wrapper

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#30]
## User story
1. As a developer
2. I want / need to check if there are negative consequences for component coupling by including headers not strictly needed outside of the class implementation
3. So that we avoid accidental coupling

## Acceptance criteria
* Document how to handle includes like this in the future
* Implement in CAN_Wrapper

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by MikeFAU** [comment]
Reviewed and discussed that the issue with the lib inside the CAN_Wrapper is planned to be considered in the Communication Layer. 

@AndreasFreund: Therefore it makes sense to keep the task open and link it with the implementation of the Communication Layer as soon as we have that. I think this leads to putting this task back into the Product Backlog.
